modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/TcpDiscoverySpi.java

```diff
@@ -2506,5 +2506,15 @@ boolean cancel() {
         @Override public void checkRingLatency(int maxHops) {
             TcpDiscoverySpi.this.impl.checkRingLatency(maxHops);
         }
+
+        @Override
+        public long getCurrentTopologyVersion() {
+            return TcpDiscoverySpi.this.ignite.cluster().topologyVersion();
+        }
+
+        @Override
```

Comment:
Please, add inherit-docs. @Override and function declaration must be at same line.